introduction.md

```diff
@@ -1,3 +1,7 @@
 Hello, im yocmen, a web dev learning mastering github
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?